_data/team.yml

```diff
@@ -2,25 +2,22 @@
 - name: Allen Meyers
   title: Storyteller
   gravatar: 900717b6b0d5430cbfb953d44cacd1de
-- name: Anthony Faber
-  title: Door Wrangler
-  gravatar: 15da69259d301b81d09f16a8ef342d62
-- name: Chuck Temple
-  title: Project Investment Fund Lead
-  gravatar: fd6fd28a3a89cd5416d47ca691a08442
-  email: chuck@codeforsanfrancisco.org
+- name: Dean Sofer
+  title: Project Success
```

Comment:
I don't think this is going to be informative for regular brigade members. Can you think of another way to communicate what you mean?

Reply:
Ummm... yeah I was trying to think of something better.

Product Manager Consultant?
Project Support?

Reply:
Project Success Consultant, maybe? Up to you!